### PR TITLE
Zipkin: support running queries with keyboard shortcut

### PR DESCRIPTION
**Pull request by olbo98**
**What this PR does / why we need it**:
This PR adds support for running queries with <kbd>Shift</kbd>+<kbd>Enter</kbd> in Explore when using Zipkin and is a complement to #37958

**Which issue(s) this PR fixes**:

Fixes # 36694

**Special notes for your reviewer**:
A plugin for running queries with <kbd>Shift</kbd>+<kbd>Enter</kbd> already exists in the QueryField component which is why it's been added to Zipkin

### CODE REVIEW

**Review by connorlindsey — APPROVED** [review]
Looks great!

**Review by ivanahuckova — APPROVED** [review]
LGTM! 

I have 1 nit pick - in `public/app/plugins/datasource/zipkin/QueryField.test.tsx` we have 1 enzyme test that you updates. I think this would be a nice opportunity to change it to react-testing-library test as we are moving from enzyme.  But it is good to go also without refactoring the test. Up to you! 🙂

File: public/app/plugins/datasource/zipkin/QueryField.test.tsx [hunk 1]
```diff
@@ -20,8 +20,8 @@ describe('QueryField', () => {
     );
 
     expect(wrapper.find(ButtonCascader).length).toBe(1);
-    expect(wrapper.find('input').length).toBe(1);
-    expect(wrapper.find('input').props().value).toBe('1234');
+    expect(wrapper.find(QueryField).length).toBe(1);
+    expect(wrapper.find(QueryField).prop('query')).toBe('1234');
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/zipkin/QueryField.test.tsx:24]: 1 more thing not directly related to your PR - in this file on line 55 we have forgotten console.log `console.log({ url });` . We could remove it.

**Comment by olbo98** [comment]
@ivanahuckova i've tried to migrate the enzyme test to react but i don't know if the tests are "enough". Could you please check it out? 🙂

File: public/app/plugins/datasource/zipkin/QueryField.test.tsx [hunk 2]
```diff
@@ -52,7 +52,6 @@ describe('useLoadOptions', () => {
           return Promise.resolve(['span1', 'span2']);
         }
 
-        console.log({ url });
```
Comment (zoltanbedi) [inline public/app/plugins/datasource/zipkin/QueryField.test.tsx:55]: 🤩

**Review by ivanahuckova — APPROVED** [review]
Looks great! Thanks for tackling that test and migrating it to react testing library. 😍

**Pull request merged**